client/src/components/LayoutComponents/Home/index.jsx

```diff
@@ -1,20 +1,44 @@
-import React from 'react';
+import React, { Component } from 'react';
+import { Paper, Typography, Grid, Box } from '@material-ui/core';
+import Styles from './style';
 import SignIn from '../../CommonComponents/SignIn/index';
-import { create } from 'jss';
-import rtl from 'jss-rtl';
-import { StylesProvider, jssPreset } from '@material-ui/core/styles';
+import { withStyles } from '@material-ui/core/styles';
 
-// Configure JSS
-const jss = create({ plugins: [...jssPreset().plugins, rtl()] });
-
-function Home() {
-	return (
-		<div>
-			<StylesProvider jss={jss}>
-				<SignIn />
-			</StylesProvider>
-		</div>
-	);
+class Home extends Component {
+	render() {
+		const { classes } = this.props;
+		return (
+			<Box width={1} mt={4}>
+				<Paper elevation={3} className={classes.paper}>
+					<Grid container xs={12} justify="flex-start" alignItems="center">
```

Comment:
I don't see the useful point about using Grid here?

Reply:
its useful especially forme when i want to do it responsible make it easier